projects/fsastorefrontlib/src/cms-components/my-account/claim/claim-details/claim-details.component.html

```diff
@@ -88,6 +88,67 @@ <h2 class="heading-headline p-3 mt-3 mb-5 text-center">
         </li>
       </ul>
     </cx-fs-accordion-item>
+    <cx-fs-accordion-item
+      *ngFor="let subclaim of claim.subclaims"
```

Comment:
Should we check first if subclaims exist on a claim? In the regular FNOL process (not replicated claim) we don't have subclaims on the claim.